src/client/sagas/repositories/index.js

```diff
@@ -15,6 +15,8 @@ export function* searchRepository(action: Object): Generator<any, any, any> {
   const response = yield call(
     fetchSaga, action.type, ALL_REPOSITORIES_QUERY, { first: action.first, filter: action.filter })
   const names = parseAllRepositoriesNames(response)
-  yield put(setRepositoriesNames(names))
+  if (names.length) {
+    yield put(setRepositoriesNames(names))
```

Comment:
probably it doesn't show it now :] but I don't like how this saga works because it dispatches event even if the error occurs, so I'll replace it with fetchAnythingSaga and add debounce support inside, and maybe merge of names... I'll take a look